easy-rsa/Windows/clean-all.bat

```diff
@@ -1,13 +1,13 @@
 @echo off
 rem move to the HOME directory specified in VARS script
-cd %HOME%
+cd "%HOME%"
 rem set a temporary KEY_DIR variable
 set d=%KEY_DIR%
 rem delete the KEY_DIR and any subdirs quietly
-rmdir /s /q %d%
+rmdir /s /q "%d%"
 rem make a new KEY_DIR
-mkdir %d%
+mkdir "%d%"
 rem copy in a fesh index file so we begin with an empty database
-copy index.txt.start %d%\index.txt
+copy index.txt.start "%d%\index.txt"
 rem copy in a fresh serial file so we begin generating keys at index 01
-copy serial.start %d%\serial.
+copy serial.start "%d%\serial."
```

Comment:
I'm not entirely sure what this line i supposed to do. Does it really just copy _serial.start_ to _serial._? What is the point of having the dot at the end of the filename?